e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -1090,9 +1092,294 @@ describeWithSnowplow("scenarios > search", () => {
 
   it("should send snowplow events for global search queries", () => {
     cy.visit("/");
-    expectGoodSnowplowEvents(PAGE_VIEW_EVENT);
     getSearchBar().type("Orders").blur();
-    expectGoodSnowplowEvents(PAGE_VIEW_EVENT + 1); // new_search_query
+    expectGoodSnowplowEvent({ event: NEW_SEARCH_QUERY_EVENT_NAME }, 1);
+  });
+
+  describe("should send snowplow events for each filter when it is applied and removed", () => {
+    describe("no filters", () => {
+      it("should not send a snowplow event when a search with no filters is accessed from the URL", () => {
+        cy.visit("/search?q=orders");
+        expectGoodSnowplowEvent({ event: NEW_SEARCH_QUERY_EVENT_NAME }, 1);
+        expectGoodSnowplowEvent({ event: SEARCH_RESULTS_FILTERED_NAME }, 0);
+      });
+    });
+
+    describe("type filter", () => {
```

Comment:
should we assert other properties too? e.g. `content_type`, `creator`, etc. All the tests below seems to only test that an `event` exist, but didn't test that they have other properties too.

Reply:
I think it's something we can add but in a different PR? we can get this one in for the PDs and then add some rigor

Reply:
sounds good.